debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -343,7 +343,7 @@ public static SecureConnectionMode parse(String value, String defaultValue) {
                                                  .withDescription("Unique name that identifies the database server and all recorded offsets, and"
                                                          + "that is used as a prefix for all schemas and topics. "
                                                          + "Each distinct MySQL installation should have a separate namespace and monitored by "
-                                                         + "at most one Debezium connector. Defaults to 'host:port'");
+                                                         + "at most one Debezium connector.");
```

Comment:
This is not a valid correction. There is no default for the field since defaults are constants and the `host:port` default value must be computed, but that computation is done in the constructor of [MySqlTaskContext](https://github.com/debezium/debezium/blob/master/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlTaskContext.java#L45).